### ISSUE TITLE
Downloaded model has errors

### ISSUE THREAD

**Issue by sebleedelisle** [paulhayes/web-3d-model-customiser#37]
I'm not entirely sure what is causing these, but even just loading in the STL into our system and saving it out without any customisation at all produces a model with errors (I hacked the code to test this). 

PrusaSlicer and Cura can auto fix these errors, you can also use Netfabb to fix them (Autodesk account required). 

The original STL file has no errors.

**Comment by paulhayes** [comment]
Shall I investigate whether openJScad V2 fixes this?

**Comment by sebleedelisle** [comment]
Yes please - a simple test would be to load the stl, and save it again without modification.
